### PR TITLE
Migrated and formatted the content from readthedocs to Hugo.

### PR DESCRIPTION
**Pull request by anvithks**
- Migrated the existing content from the `readthedocs` folder to the `content/` folder as per Hugo.  
- Modified the files to add front matter as per Hugo.  
- Formatted the files for consistency.  
- Created the following sections:  
   * Introduction  
   * Concepts  
   * Community  
   * Tutorials and Guides  
   * Official Releases
- Fixed typos in image file names.

### CODE REVIEW

File: content/guides/developer-guides/orchestration/_index.md [hunk 1]
```diff
@@ -134,13 +147,16 @@ parameters:
...
-### Points to remember:
-    a) For every action/task, there should be an entry point. entry_point is the script to execute this action. This can either be a python script or a shell script. OpenSDS prefers python script for using the rich python libraries, as required
-    b) All the parameters of this actions are detailed here. It should define the 'type' of the parameter, whether this parameter is 'required' or not and if the pramaeter can have some 'default' value
-    c) The action should specify 'runner_type'. Ex: runner_type: "python_script"
+```  
+
+### Points to remember  
+
+1. For every action/task, there should be an entry point. entry_point is the script to execute this action. This can either be a python script or a shell script. OpenSDS prefers python script for using the rich python libraries, as required
+2. All the parameters of this actions are detailed here. It should define the '`type`' of the parameter, whether this parameter is '`required`' or not and if the pramaeter can have some '`default`' value
```
Comment (joseph-v) [inline content/guides/developer-guides/orchestration/_index.md]: Quotations are not required if we are using code highlighting. (repeated in several places)

Comment (anvithks) [reply content/guides/developer-guides/orchestration/_index.md]: I will remove the quotations.

Comment (anvithks) [reply content/guides/developer-guides/orchestration/_index.md]: @joseph-v  This is fixed in the latest commit.
You can verify and resolve this.

**Comment by anvithks** [comment]
@skdwriting Please review and merge.

**Comment by joseph-v** [comment]
LGTM

**Comment by anvithks** [comment]
> 1. Please check https://docs.google.com/document/d/1EFyzeH0Jvre0R8_uu4OUcjE6DlDcolxfAbPNLxusrak/edit for layout points and update the layout
> 2. Features page : Please provide the intro and links to existing guides.

I will follow the document and make the appropriate changes.

**Pull request merged**